### ISSUE TITLE
onFinishRating always returns initial value

### ISSUE THREAD

**Issue by tfsjohan** [Monte9/react-native-ratings#24]
When clicking a star I always get the same score, regardless of what star I press. What am I doing wrong?

(In this case, score is always 2 in onFinishRating)

```
    <Rating
                            type={'star'}
                            imageSize={20}
                            ratingCount={5}
                            startingValue={2}
                            onFinishRating={(score: number) => this.rank(item, score)}/>
```

**Comment by neilzhengzx** [comment]
I get the same wrong.

**Comment by Monte9** [comment]
Yes, that's right. For now.. sliding & clicking is a missing feature. 

If someone has some time, please submit a PR for it. Thanks

**Comment by smemamian** [comment]
> Yes, that's right. For now.. sliding & clicking is a missing feature for the `<Rating />` component.
> 
> If you are only interested in the click feature.. then you can use the `<AirbnbRating />` component which has simple animations too.
> 
> If someone has some time, please submit a PR for it. Thanks

but when you are using `AirbnbRating` you missing the touch rating feature.
